webapp/lib/app/nook/view.dart

```diff
@@ -33,8 +35,10 @@ class NookPageView extends PageView {
   Map<TagFilterType, ConversationFilter> conversationFilter;
   ConversationPanelView conversationPanelView;
   ReplyPanelView replyPanelView;
+  NotesPanelView notesPanelView;
   TagPanelView tagPanelView;
```

Comment:
nit: let's swap these round so replies and tags can be next to each other
```suggestion
  TagPanelView tagPanelView;
  NotesPanelView notesPanelView;
```